### ISSUE TITLE
[Preview-MAS Violation]:Screen reader is narrating play button info twice when focus it.

### ISSUE THREAD

**Issue by NavyaGoberu** [nvaccess/nvda#9723]
User Experience:
User who rely on screen reader if user select any button and it is narrating twice of button info user will confuse.

Test Environment:
OS: Windows 10
Browser: IE
URL: https://od.marketingsites-ppe.microsoft.com/en-us/enterprise/onegdc_video?preview=inprogress&variationChoices=*:default

Repro Steps:
Step 1: Open the above URL using any browser.
Step 2: Navigate to section “primaryArea” and press tab to select play buttons. 
Step 3 : Verify screen reader is narrating aria-label value or not?

Actual:
Screen reader is narrating play button info twice on normal navigation and reverse navigation it is narrating single time but it’s not narrating role of the play.
Expected:
Screen reader should be narrate the play button info single time with role attribute.

MAS Reference:
MAS 4.1.2
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Comment by Adriani90** [comment]
@NavyaGoberu for this issue to be reproduced someone needs a mandant to the Microsoft which is not possible to gather for everyone. So please contact NV Access directly and give them the correct permissions to access the app and to reproduce this issue. It is not possible to investigate this by the community. I am closing this for now.
If you like, you can contact NV Access at info@nvaccess.org
